rtsp-to-web/Dockerfile

```diff
@@ -1,47 +1,47 @@
 # syntax=docker/dockerfile:1
 #
 # Example build command from add-on root directory:
-# $ docker build --build-arg BUILD_FROM="homeassistant/amd64-base:latest" --build-arg "BUILD_ARCH=amd64" --build-arg TEMPIO_VERSION=2021.09.0 -t rtsp-to-web rtsp-to-web
-
-ARG BUILD_FROM
+# $ docker build --tag rtsp-to-web rtsp-to-web
 
 # https://developers.home-assistant.io/docs/add-ons/configuration#add-on-dockerfile
-FROM $BUILD_FROM
-ARG TEMPIO_VERSION=2021.09.0
-ARG BUILD_ARCH
-
-# Execute during the build of the image
-RUN \
-    curl -sSLf -o /usr/bin/tempio \
-    "https://github.com/home-assistant/tempio/releases/download/${TEMPIO_VERSION}/tempio_${BUILD_ARCH}"
+ARG BUILD_ARCH="amd64"
+ARG BUILD_FROM="ghcr.io/hassio-addons/base/${BUILD_ARCH}:11.0.1"
 
 
-RUN apk add go git
+FROM ${BUILD_FROM} AS build
 
-ARG REPO=github.com/deepch/RTSPtoWeb
-WORKDIR /build/$REPO
+RUN apk add --no-cache git go=~1.17.4-r0
 
-RUN git clone https://$REPO .
-RUN git fetch
-RUN git reset --hard "64289619ffa24d99ca3e0537fe37f2660cb79cf6"
+# Clone repo
+ARG REPO="deepch/RTSPtoWeb"
+WORKDIR /workspace
+RUN \
+    git clone https://github.com/$REPO . \
+    && git reset --hard "64289619ffa24d99ca3e0537fe37f2660cb79cf6"
```

Comment:
Just an FYI, I have permissions to make RTSPtoWeb releases so we can start marking versions there and remove this pin to a specific commit https://github.com/deepch/RTSPtoWeb/issues/91

Reply:
That will be nice!